README.md

```diff
@@ -99,3 +99,12 @@ plugins {
 ### Requirements
 
 Gradle 5.3.1 and JDK 8 are required.
+
+
+You can override checkstyle version using a dedicated configuration (default is 8.24):
+```kotlin
+checkstyle {
+    toolVersion = "8.20"
+    configFile = file("${project.rootDir}/config/checkstyle.xml")
```

Comment:
Hi,
Thanks for the PR.
Can you please add a note about the fact that the default `checkstyle.xml` is compatible with `8.24` or later.

Reply:
Hi, 
Yes i think it's a good idea.